programs/cw-ics08-wasm-eth/src/contract.rs

```diff
@@ -140,6 +145,27 @@ pub fn query(
     }
 }
 
+/// The migrate entry point for the CosmWasm contract.
+/// # Errors
+/// Will return an errror if the state version is not newer than the current one.
+#[entry_point]
+#[allow(clippy::needless_pass_by_value)]
+pub fn migrate(
+    deps: DepsMut<EthereumCustomQuery>,
+    _env: Env,
+    _msg: MigrateMsg,
+) -> Result<Response, ContractError> {
+    // TODO: REMOVE
+    if cw2::CONTRACT.may_load(deps.storage)?.is_none() {
+        cw2::set_contract_version(deps.storage, CONTRACT_NAME, STATE_VERSION)?;
+    }
```

Comment:
You shouldn't need this. If your worry is the testnet contract, just add this locally. Seems easier than removing in another PR

Reply:
I just removed it ^^